openquake/commonlib/source.py

```diff
@@ -536,6 +554,17 @@ def __repr__(self):
 ])
 
 
+def cjoin(strings):
+    """
+    :param strings: an iterable over strings
+    :returns: a comma separated string of lenght <= LENGHT
+    """
+    s = ','.join(strings)
+    if len(s) > LENGTH:
+        raise ValueError('The string %r is over %d characters' % (s, LENGTH))
```

Comment:
Please add a comment to explain why a string longer than `LENGTH` would cause problems, and in what kind of situation it might happen.